src/ofxLSLReceiver.h

```diff
@@ -75,6 +80,7 @@ class Receiver : public BaseReceiver {
     sampleBuffer->timeStamp = ts;
     if (ts > 0) {
       std::lock_guard<std::mutex> lock(pullMutex);
+	  sampleBuffer->timeCorrection = getTimeCorrection(1);  // update the time correction value when pulling sample
```

Comment:
According to the library, 
> @param timeout Timeout to acquire the first time-correction estimate (default: no timeout).
Looks like this timeout is only for the first correction. So it is better to remove the timeout all together, since it only affects the first reading.

Reply:
Tested working. If the stream is lost, it tries to reconnect to the stream.